app/grandchallenge/github/migrations/0008_auto_20220201_1203.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.10 on 2022-02-01 12:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("github", "0007_auto_20220125_1411"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="githubwebhookmessage", name="has_open_source_license",
+        ),
+        migrations.AlterField(
+            model_name="githubwebhookmessage",
+            name="clone_status",
+            field=models.CharField(
+                choices=[
+                    ("PENDING", "Pending"),
+                    ("STARTED", "Started"),
+                    ("SUCCESS", "Success"),
+                    ("FAILURE", "Failure"),
+                    ("INVALID", "Invalid"),
+                    ("NOT_APPLICABLE", "Not Applicable"),
+                ],
+                default="NOT_APPLICABLE",
+                max_length=14,
+            ),
+        ),
+        migrations.RemoveField(
+            model_name="githubwebhookmessage", name="license_check_result",
```

Comment:
Note that I am removing the old field here and adding the JSON one back, will re-run the check on the existing messages.